### DISCUSSION TITLE
[A-L] (10 Febbraio 2023) Esercizio 5

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#376] [Esami A-L]
Sia R una relazione binaria su un insieme $\Large A$ tale che per ogni $\Large a \in A$ esiste $\Large b \in A$ tale che $\Large (a, b) \in R$.

Indicare se le seguenti affermazioni sono vere o false.

1. R è transitiva.

2. Se R è simmetrica e transitiva allora R è riflessiva.

3. Se R è riflessiva e simmetrica allora R è transitiva.

**Comment by cek753** [comment]
1.Falso
2.Vero
3.Falso
